CondFormats/EcalObjects/interface/EcalChannelStatusCode.h

```diff
@@ -17,25 +17,13 @@
 
 class EcalChannelStatusCode {
 public:
-  enum Code {
-    kOk = 0,
-    kDAC,
-    kNoLaser,
-    kNoisy,
-    kNNoisy,
-    kNNNoisy,
-    kNNNNoisy,
-    kNNNNNoisy,
-    kFixedG6,
-    kFixedG1,
-    kFixedG0,
-    kNonRespondingIsolated,
-    kDeadVFE,
-    kDeadFE,
-    kNoDataNoTP
-  };
+  typedef int Code;
+  static constexpr int kOk = 0, kDAC = 1, kNoLaser = 2, kNoisy = 3, kNNoisy = 4, kNNNoisy = 5, kNNNNoisy = 6,
+                       kNNNNNoisy = 7, kFixedG6 = 8, kFixedG1 = 9, kFixedG0 = 10, kNonRespondingIsolated = 11,
+                       kDeadVFE = 12, kDeadFE = 13, kNoDataNoTP = 14;
 
-  enum Bits { kHV = 0, kLV, kDAQ, kTP, kTrigger, kTemperature, kNextToDead };
+  typedef int Bits;
+  static constexpr int kHV = 0, kLV = 1, kDAQ = 2, kTP = 3, kTrigger = 4, kTemperature = 5, kNextToDead = 6;
```

Comment:
Nothing in #44487 seemed to use these, so maybe it would be better to keep it as `enum` too?

Reply:
OK.